lib/ansible/modules/packaging/os/pacman.py

```diff
@@ -138,21 +138,62 @@ def get_version(pacman_output):
     return None
 
 
+def find_provider(module, pacman_path, target_package):
+    """Find the package providing `target_package`"""
+    cmd = "%s -Q %s" % (pacman_path, target_package)
```

Comment:
```suggestion
    cmd = "%s --query %s" % (pacman_path, target_package)
```
I personally prefer if we use the long options.